src/common/commonutils/DeviceInfoUtils.c

```diff
@@ -328,7 +328,7 @@ bool CheckCpuFlagSupported(const char* cpuFlag, char** reason, void* log)
     bool result = false;
     char* cpuFlags = GetCpuFlags(log);
 
-    if ((NULL != cpuFlag) && (NULL != strstr(cpuFlags, cpuFlag)))
+    if ((NULL != cpuFlag) && (NULL != cpuFlags) && (NULL != strstr(cpuFlags, cpuFlag)))

```

Comment:
>(NULL != cpuFlags) && (NULL !=  [](http://example.com/codeflow?start=29&length=31)

On what distro did you find the lscpu command to not be present or to fail? Was that a default original configuration of that distro, or was something done to break it? Can you please give more details about how this bug was found.